doc/swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Platform API",
```

Comment:
maybe `Publishing Platform API`?

Reply:
This entity, `Platform`, is not only used for publishing. It is also used in `taoSync` and as I understand it should/will be moved to `tao-core` in the future